inkcpp_compiler/emitter.h

```diff
@@ -2,6 +2,7 @@
 
 #include "command.h"
 #include "system.h"
+#include "version.h"
```

Comment:
Looks like you don't actually use this header here? You can just include it in the relevant cpp file.